lib/core/src/lib/context-menu/context-menu.directive.ts

```diff
@@ -27,7 +27,7 @@ import { ContextMenuOverlayService } from './context-menu-overlay.service';
 export class ContextMenuDirective {
     /** Items for the menu. */
     @Input('adf-context-menu')
-    links: any[];
+    getActions: () => any[];
```

Comment:
Are you sure this directive is not exported? Otherwise, this would be an unnecessary breaking change.

Reply:
The directive is exported. What would be your suggestion? I needed to adjust the return type to make sure we set the actions when right before the context menu is opened, like it was requested in the ticket.